src/pages/Applicant/EditableProfile/EditableDetail/container.tsx

```diff
@@ -107,23 +95,21 @@ const EditableDetailContainer: FunctionComponent = () => {
 
   if (applicantError || translationsError) history.push(RoutesBuilder.notFound);
```

Comment:
this line doesn't feels right. Hay que discutir un poco que hacemos

Reply:
![image](https://user-images.githubusercontent.com/20487262/78406222-92881900-75d8-11ea-9da9-3547bc3bf1fe.png)

estoy de acuerdo, lo habiamos hablado igual, que para cosas que no interrumpen el flow y form submissions tirar una alertita, y para errores que interrumpen el flow (500 o 404 en el getapplicant) redirigir a pantallas de error